retreaver-typescript/src/base/Request.ts

```
@@ -0,0 +1,265 @@
+
+  /**
+   * Request JSON data from the host
+   * @param requestUrl - The request URI
+   * @param payload - Post object
+   * @param callbacks - Array or Function to be called after request
+   * @param context - Context applied to callback
+   */
+  getJSON<T = any>(
+    requestUrl: string,
+    payload?: any,
+    callbacks?: Function | Function[],
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)